templates/content/node--oe-news--teaser.html.twig

```diff
@@ -10,7 +10,8 @@
   'url': url,
   'title': content.extra_field_oe_theme_helper_short_title_with_fallback|field_value,
   'meta': [
-    content.oe_publication_date|field_value
```

Comment:
Why you removed this? This was ok, we need to move to use drupal fields now that we export the teaser view mode.